composer.json

```diff
@@ -63,7 +63,6 @@
         }
     },
     "support": {
-        "email": "cakephp+csvview@josediazgonzalez.com",
```

Comment:
If he wants to share his email leave it be :)

Reply:
I should really just remove my email from packages or change it to `email-for-consulting@josediazgonzalez.com`...